thrifty-kotlin-codegen/src/test/kotlin/com/microsoft/thrifty/kgen/util.kt

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.thrifty.kgen
+
+import com.squareup.kotlinpoet.FileSpec
+import io.kotest.matchers.compilation.shouldCompile
+
+fun List<FileSpec>.shouldCompile() {
+    forEach {
+        it.shouldCompile()
+    }
+}
+
+fun FileSpec.shouldCompile() {
```

Comment:
nit: we may as well put these in `KotlinCodeGeneratorTest.kt` since that's the only place they're used.  We can pull these out again if/when we need to.

Reply:
sure, done